doc/manual/cookbooks/initial-condition-S20RTS/S20RTS.part.prm.out

```diff
@@ -5,10 +5,9 @@ subsection %%\hyperref[parameters:Initial_20conditions]{Initial conditions}%
     set %%\hyperref[parameters:Initial conditions/S40RTS perturbation/Initial condition file name]{Initial condition file name}%       = S20RTS.sph%% \index[prmindex]{Initial condition file name} \index[prmindexfull]{Initial conditions!S40RTS perturbation!Initial condition file name} %
     set %%\hyperref[parameters:Initial conditions/S40RTS perturbation/Spline knots depth file name]{Spline knots depth file name}%      = Spline_knots.txt%% \index[prmindex]{Spline knots depth file name} \index[prmindexfull]{Initial conditions!S40RTS perturbation!Spline knots depth file name} %
     set %%\hyperref[parameters:Initial conditions/S40RTS perturbation/Remove degree 0 from perturbation]{Remove degree 0 from perturbation}% = false%% \index[prmindex]{Remove degree 0 from perturbation} \index[prmindexfull]{Initial conditions!S40RTS perturbation!Remove degree 0 from perturbation} %
-    set %%\hyperref[parameters:Initial conditions/S40RTS perturbation/Vs to density scaling]{Vs to density scaling}%             = 0.2 %% \index[prmindex]{Vs to density scaling} \index[prmindexfull]{Initial conditions!S40RTS perturbation!Vs to density scaling} %
+    set %%\hyperref[parameters:Initial conditions/S40RTS perturbation/Vs to density scaling]{Vs to density scaling}%             = 0.15 %% \index[prmindex]{Vs to density scaling} \index[prmindexfull]{Initial conditions!S40RTS perturbation!Vs to density scaling} %
     set %%\hyperref[parameters:Initial conditions/S40RTS perturbation/Thermal expansion coefficient in initial temperature scaling]{Thermal expansion coefficient in initial temperature scaling}% = 3e-5%% \index[prmindex]{Thermal expansion coefficient in initial temperature scaling} \index[prmindexfull]{Initial conditions!S40RTS perturbation!Thermal expansion coefficient in initial temperature scaling} %
     set %%\hyperref[parameters:Initial conditions/S40RTS perturbation/Reference temperature]{Reference temperature}%             = 1600%% \index[prmindex]{Reference temperature} \index[prmindexfull]{Initial conditions!S40RTS perturbation!Reference temperature} %
-    set %%\hyperref[parameters:Initial conditions/S40RTS perturbation/Remove temperature heterogeneity down to specified depth]{Remove temperature heterogeneity down to specified depth}% = 200000%% \index[prmindex]{Remove temperature heterogeneity down to specified depth} \index[prmindexfull]{Initial conditions!S40RTS perturbation!Remove temperature heterogeneity down to specified depth} %
```

Comment:
Okay, good. I guess we can merge this one then, but we should look into what happened in #1120. As you reviewed that, do you have an idea what the problem is?